templates/kafka/0/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+zk:
```

Comment:
Is there a reason this needs a copy of its own zookeeper, apparently based on ours, instead of having a question of type service to allow the user to point to their own (possibly already existing) stack?